lib/ts/BitSetListInit.h

```diff
@@ -0,0 +1,45 @@
+/** @file
```

Comment:
I think this is overkill. It's only used once in the base code and if used almost exclusively in one test file it should be embedded there and not a global header. For the one use in main code, that should probably be a static with a hand tuned initializer - the code is simple enough you could probably use a local lambda to do it if the initializer form is sufficiently desired for robustness.